mav-sdk/examples/geofence.rs

```diff
@@ -0,0 +1,50 @@
+use mav_sdk::Drone;
+
+const MAVSDK_SERVER: &str = "http://127.0.0.1:4000";
+
+#[tokio::main]
+async fn main() {
+    let mut drone = Drone::connect(MAVSDK_SERVER).await.expect("Should connect");
+    println!("We have a DRONE connection with {}", MAVSDK_SERVER);
+
+    println!("Uploading Geofence...");
+
+    let upload_geofence_request = mav_sdk::grpc::geofence::UploadGeofenceRequest { polygons: vec![
+        mav_sdk::grpc::geofence::Polygon {
+            points:vec![mav_sdk::grpc::geofence::Point{ latitude_deg: 0.0, longitude_deg: 0.0 },mav_sdk::grpc::geofence::Point{ latitude_deg: 0.0, longitude_deg: 0.0 },mav_sdk::grpc::geofence::Point{ latitude_deg: 0.0, longitude_deg: 0.0 },mav_sdk::grpc::geofence::Point{ latitude_deg: 0.0, longitude_deg: 0.0 },], fence_type: 0, },
```

Comment:
Ok, that's a great start. could we just add actual coordinates here?

- [ ] Let's use the actual GPS coordinates of the drone (at the moment) and update the Polygon accordingly.
E.g. `current coordinates +/- a value in degrees` in all directions to get a rectangle with a given size. Keep in mind that we work with degrees here so that we can define the distance in degrees, instead of size (i.e. ~0.25 degrees instead of ~28 km). We do need to choose a valid value as "Each degree of latitude is approximately 111 kilometres apart" (https://www.thoughtco.com/degree-of-latitude-and-longitude-distance-4070616). 

- [ ] Import `Point`, `UploadGeofenceRequest`, `ClearGeofenceRequest`